gui/services/migrations/0195_auto__add_field_nfs_nfs_srv_mountd_log__add_field_nfs_nfs_srv_statd_lo.py

```diff
@@ -11,7 +11,7 @@ def forwards(self, orm):
         # Adding field 'NFS.nfs_srv_mountd_log'
         db.add_column(u'services_nfs', 'nfs_srv_mountd_log',
                       self.gf('django.db.models.fields.BooleanField')(default=True),
-                      keep_default=False)
+                      keep_default=True)
 
```

Comment:
No, thats not what I told you to change, read again please.
Also you msut change it for both columns.